components/ai_chat/resources/untrusted_conversation_frame/components/assistant_response/index.tsx

```diff
@@ -103,6 +106,11 @@ function AssistantEvent(props: BaseProps & {
     )
   }
   if (props.event.toolUseEvent) {
+    if (props.event.toolUseEvent.toolName === Mojom.MEMORY_STORAGE_TOOL_NAME) {
```

Comment:
Please add an example toolUseEvent to storybook via `components/ai_chat/resources/page/stories/components_panel.tsx` so we can preview and more easily adjust styles there.